packages/nodejs/project.bri

```diff
@@ -63,20 +135,38 @@ export async function test(): Promise<std.Recipe<std.File>> {
 
 export function liveUpdate(): std.WithRunnable {
   const src = std.file(std.indoc`
```

Comment:
This script got kinda complicated! Might be worth pulling this out into a separate `.nu` file

I'm pretty happy with how the script works though: it automatically updates the list of major versions (including new major versions), and sets the latest version within each major version.